src/components/templates/Auditorium/components/Section/Section.scss

```diff
@@ -0,0 +1,105 @@
+@import "scss/constants";
```

Comment:
This file seems to be mostly if not literally a duplicate of `src/components/templates/Audience/Audience.scss`

It would be good to de-dupe these, or at the very least add some `@debt` comments explaining why it was copy/paste cloned, etc for future reference/knowledge/etc.

Reply:
It was copy/pasted + renamed. I think the styles work good enough in Audience and I copy-pasted them. The classnames are different though, so they will not interfere. As I described above, I hope to make Audience legacy as soon as this new Auditorium is out.

Reply:
Mm, fair enough. I guess I am just wary of 'best intentions' based on past experience; and for the sake of adding 2 comments explaining the duplication (even if they get deleted along with the refactor PR later on), we can avoid getting into this 'duplicated by noone actually knows why' situation the codebase is currently plagued by.

Reply:
added in https://github.com/sparkletown/sparkle/pull/1251/commits/4525887a96f74fe7e8a1e8316d75e0aa3946aac9